### PR TITLE
YDMU Teferi's Contingency and Darigaaz's Whelp (WIP)

### PR DESCRIPTION
**Pull request by Simisays**
Having difficulty scripting these two. It's always a pain to find the correct rememberobject for these perpetual cards

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/darigaazs_whelp.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Name:Darigaaz's Whelp
+ManaCost:1 B
+Types:Creature Dragon
+PT:2/1
+K:Kicker:1 R
+K:Flying
+T:Mode$ Drawn | ValidCard$ Card.YouOwn+Dragon | TriggerZones$ Battlefield | RememberDrawn$ True | Execute$ DBEffect | TriggerDescription$ Whenever you draw a Dragon card, it perpetually gets +1/+1.
+SVar:DBEffect:DB$ Effect | StaticAbilities$ PerpetualP1P1 | RememberObjects$ Remembered | Name$ Darigaaz's Whelp's Perpetual Effect | Duration$ Permanent
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/darigaazs_whelp.txt]: why do we need RememberDrawn, can't we just use some Triggered value instead?

**Comment by Hanmac** [comment]
how does the Darigaaz's Whelp works on Arena? does Opponent see the Dragon Triggering? and can opponent Response to it?

**Comment by Simisays** [comment]
> how does the Darigaaz's Whelp works on Arena? does Opponent see the Dragon Triggering? and can opponent Response to it?

Hmm good point, I'll boot up Arena and test it when I get home from work. Though normally you can respond to the giving of perpetual effects

**Comment by Hanmac** [comment]
also the problem with such Perpetual Effects is that for our GUI we still show the Remembered Cards (so opponent knows what card was drawn?)
Would The Trigger show it too with the Triggered Objects in the Stack Description?

**Comment by Hanmac** [comment]
@Simisays is there a reason why you deleted the branch?

**Comment by Simisays** [comment]
Yeah I messed up. Tried renaming the branch because I have a branch with the same name on my PC. Guess I learned something today. Will create a new PR

**Pull request closed without merging**